features/facebookImport/src/model/analyses/ministories/advertising-value-analysis.js

```diff
@@ -28,14 +31,23 @@ export default class AdvertisingValueAnalysis extends RootAnalysis {
         }
         this._randomAdInterests = [...randomAdInterests];
         this._numberInterests = numberInterests;
+        this._displayData = adInterests.sort((a, b) => (a > b ? 1 : -1));
```

Comment:
Do you need to add the sort routine here? If you want the reverse order, just use [reverse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse) after `sort`

Reply:
Actually not what you're asking for here, but I forgot to add lowerCase-logic, so I changed the function a bit in the latest push